common/src/api/external/mod.rs

```diff
@@ -1179,6 +1185,9 @@ pub struct Instance {
 
     #[serde(flatten)]
     pub runtime: InstanceRuntimeState,
+
+    /// The cooldown period between automatic restarts of this instance.
+    pub auto_restart_cooldown: std::time::Duration,
```

Comment:
https://github.com/oxidecomputer/omicron/pull/6503/commits/308d3e96ac481f1c1472ab447a2f6ab5f2b07db9 changed the params type to just a number of seconds (because asking for a cooldown with nanoseconds precision seemed pretty silly). I could also make the type in the external API be seconds, as it's just intended to be consumed by UIs and stuff.

Reply:
Secondarily...yeah, there should probably be a minimum. Maybe a minute?

Reply:
Made both those changes in https://github.com/oxidecomputer/omicron/pull/6503/commits/a0d08c500c8995fac4ec44bcde6cc461a8e08799.